gatsby-node.js

```diff
@@ -115,13 +117,130 @@ exports.sourceNodes = async ({ actions }) => {
   return
 }
 
-// exports.onCreateNode = ({ actions, node }) => {
-//   const { type } = node.internal
-//   if (type !== `Member`) return
-//   const { bio } = node
-//   const bioAsHtml = markdownProcessor.process(bio)
-//   node.bio = bioAsHtml
-// }
+exports.onCreateNode = ({ node, actions }) => {
+  const { type } = node.internal
+  if (type === 'GhostAuthor') {
+    if (node.slug !== 'data-schema-author') {
+      actions.createNodeField({
+        node,
+        name: `member___NODE`,
```

Comment:
Can you include a comment with the webpages which document this approach? other ppl will have not much of an idea of what's going on when they look over dis code